conformance/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
I'm not sure about overriding the default logging configuration of spark. 
@lokm01 @yruslan thoughts?

Reply:
if this file ins not present, then `Logger` used in `TimeZoneNormalizer` fires warning on start-up.